lnwallet/test/test_interface.go

```diff
@@ -2963,7 +2963,9 @@ func testSingleFunderExternalFundingTx(miner *rpctest.Harness,
 	// we'll create a new chanfunding.Assembler hacked by Alice's wallet.
 	aliceChanFunder := chanfunding.NewWalletAssembler(
 		chanfunding.WalletConfig{
-			CoinSource:            lnwallet.NewCoinSource(alice),
+			CoinSource: lnwallet.NewCoinSource(
+				alice, nil,
```

Comment:
suggestion: rather than giving a nil argument here, can I suggest we instead have a closure that always returns true for clarity?

```go
allowAll := func(_ Utxo) bool { return true }
```

In general I'd like to discourage `nil` anything if it can be avoided.